### ISSUE TITLE
Enhance Table Designer with Dragging and MultiSelect

### ISSUE THREAD

**Issue by ShadowFiendZX** [microsoft/azuredatastudio#18745]
This is sort of similar to [this](https://github.com/microsoft/azuredatastudio/issues/18717) open issue.
Like the linked issue, it would be nice to insert in place, however the table designer functionality could use a bit of enhancement.

Two features I think would be useful in ADS are :

1) Allow the reorder of columns by dragging on them on the left side
2) Allow multiple columns to be selected at once to either move, delete or copy & paste to a different table designer window.

These features are in SSMS, and would make ADS more useful, as the designer in SSMS is blindingly white whereas ADS has dark mode.

**Comment by ShadowFiendZX** [comment]
IE
Let's say you work at a company that has policies about the order of fields, namely fields that 90% of the tables have and are used by an application. If someone adds them in the "wrong" order, SSMS allows you to easily change the order of the fields without having to manually write a script to select the data into a temp table, drop the main table, then select the data out of the temp table and into the newly created table, etc.

In this case, you have C# representations of tables, which you might typically define/update before creating/updating the table. If you have policies where the class and table should define the columns in the same order, and your C# representation is the desired order, changing the order within the table designer would make the process a lot easier.

**Comment by erinstellato-ms** [comment]
@ShadowFiendZX Ok.  Note that even if you change the order, the same process occurs behind the scenes (which also happens in SSMS now): a temp table is created, data inserted into it, and the old table dropped, new table renamed.  You want to have ADS handle the scripting.

**Comment by ShadowFiendZX** [comment]
Yes, I just think it's more reliable/convenient if the program handles it behind the scenes, not to mention faster since the fastest way I could think of doing it manually would be to right click -> script as, then copy and paste/edit the resulting script to the desired sequence of scripts to run.

**Comment by alanrenmsft** [comment]
@ShadowFiendZX we have implemented the reordering feature, please try it out with latest ADS release. we will use this issue to track the multiple selection feature request.
